### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by TimmyChan99**
Build a simple HTML list of To-Do tasks

- Use Webpack.
- Tasks list loads dynamically.
- Create an array of objects containing the tasks and a function that iterates over the tasks.

### CODE REVIEW

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @TimmyChan99,

Good job so far!
✔️ Nice use of `sort()` method
✔️ Use of README template
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Kindly update your README with the correct project title and add a paragraph about the project.
- [ ] In the **Getting Started** section of your README kindly include the correct `git clone` link. Also include `npm install` and `npm start` (or `npm run build`) as necessary commands for running the project locally.
- [ ] For proper documentation, kindly remove the dummy second **Author** since this is a solo project

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -25,9 +25,8 @@ To get a local copy up and running follow these simple example steps.
...
 - Run `git clone https://github.com/TimmyChan99/To_Do_list.git`
 - Go to the folder `cd To_Do_list`
-- Install packages run `npm install`
-- To run application using webpack-dev-server run `npm start`
-
+- Install packages: run `npm install`
+- To run application using webpack-dev-server: run `npm start`
 
 
 ## Authors
```
Comment (brytebee) [inline README.md:32]: - [ ] **Required:** To visitors especially recruiters see what your application looks like, ensure your screenshot is working correctly. At the moment, the link is broken.
image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @TimmyChan99 👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (TimmyChan99) [reply README.md:32]: I have checked and the screenshot works fine on my PC and other devices
Screenshot 2022-01-12 094423

**Comment by TimmyChan99** [comment]
I have checked and the screenshot works fine on my PC and other devices
Screenshot 2022-01-12 094423
**Please check the project properly and be sure if there is an issue before submitting the review.**

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @TimmyChan99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**